base/docs/helpdb/Base.jl

```diff
@@ -4057,7 +4057,8 @@ abs2
     write(filename::AbstractString, x)
 
 Write the canonical binary representation of a value to the given I/O stream or file.
-Returns the number of bytes written into the stream.
+Returns the number of bytes written into the stream. *`write(::IOStream, ...)` is a synchronous operation and will block at the process level.*
```

Comment:
Also, it's a bit weird to say that only the `::IOStream` method is blocking, since this also applies to the `::AbstractString` one. Why not just say "This is a ...". Same below.